config/locales/en.yml

```diff
@@ -436,7 +436,9 @@ en:
         under_10: "Enter a total charge that is at least £10.00 per week"
         less_than_shortfall: "The total charge must be more than the outstanding amount"
       rent_period:
-        invalid_for_org: "%{org_name} does not use %{rent_period} as a rent period. Choose another rent period, or a data coordinator can add rent periods to your organisation"
+        invalid_for_org:
+          period: "%{org_name} does not use %{rent_period} as a rent period. Choose another rent period, or a data coordinator can add rent periods to your organisation"
+          managing_org: "%{org_name} does not use %{rent_period} as a rent period. Set another rent period on this log, or a data coordinator can add rent periods to this organisation"
```

Comment:
Felt like it needed slightly different wording to the one on the period question but there may be better options than this.